projects/managed-hubs/sales.md

```diff
@@ -1,96 +1,99 @@
+If you wish to help with sales and growing the service, please send an email to `partnerships@2i2c.org` so you can be added to that list, and be given a quick run-down of the process.
 
-Status fields indicate each step of the sales process
-: We use Monday **status fields** to indicate where we are in the leads process. For example, a new lead is in the "Send Questionnaire" status of the "Triage Phase". This means that the next action is to send them a questionnaire.
-
-Fill in information following each status column
-: As leads progress through the sales process, we need more information from them, eventually leading to a contract. Fill in this information, and once we reach the next "status" column on the board, we have entered a new phase of the process. At the end of a sale (AKA, once an invoice is sent an paid), all columns of the row should be filled.
+All members of `partnerships@2i2c.org` are invited to participate in the process below.
 
-### Triage stage
+## Tracking leads with AirTable
 
-The first stage is to understand whether an interested party is in-scope for 2i2c, and a good fit for the Managed Hub Service.
+We use AirTable to track our leads for prospective communities to serve.
+: [**This AirTable**](https://airtable.com/appDUEjjcSqHfilFD/tblxLUQ3ElIaaQuM8/viwXstuM3K48smZXW?blocks=hide) keeps track of all potential communities we may serve. Below is an overview of each step in the process.
 
-Create a lead
-: When others show obvious interest in working with 2i2c, they are now a lead. [**Create a new item in the sales pipeline**](https://icsi-company.monday.com/boards/1140216585) and fill in the relevant information for them (everything up to "triage status").
+Status fields indicate each step of the sales process
+: We use the **status column** to indicate where a community is in the process. There are roughly five statuses that a community can have:
 
-Send the questionnaire
-: To understand whether a lead is a good fit, we've created [a short questionnaire](https://forms.gle/VFzs364iNrJgKnKu9) for them. Send a link to the questionnaire.
+  - {guilabel}`Active Lead` is a community that is actively interested in our service.
+  - {guilabel}`Inactive Lead` is a potential community that is less-responsive or does not seem interested anymore.
+  - {guilabel}`Needs Admin` is a lead that has agreed to a service tier and now needs CS&S to set up administrative infrastructure.
+  - {guilabel}`Active Service` is a community for which we have an active service contract.
+  - {guilabel}`Inactive Service` is a community that we used to run a service for, but who has since stopped that service.
+  - {guilabel}`Waiting` is a community that is interested in our service, but we are intentionally waiting for an extended period of time to follow-up (for example, if they are waiting to learn if they have received a grant).
```

Comment:
I see two dimensions on these statuses... 
One dimension:
Lead > Waiting > Admin > Service
And the other one:
Active > Inactive
Is this how you are envisioning the pipeline?

Reply:
I think that's right - my thinking is that it's useful if we know whether an organization went from "Lead -> Inactive" or from "Service -> Inactive"

are you suggesting that this would be better-described by 2 metadata fields (one for category, one for active/inactive)?

Reply:
> are you suggesting that this would be better-described by 2 metadata fields (one for category, one for active/inactive)?

Yep, I think so... otherwise you are "corrupting" a "natural" pipeline with a different level of status...

Reply:
Would you be OK with us merging as-is, and we can discuss the right pipeline to use for this process in the sustainability team meetings? I think the pipeline should evolve, but this PR at least updates the process so that it reflects our current practice.

Reply:
That would be OK to me if we follow up later.